### ISSUE TITLE
Incorrect no-member warnings returned from struct.calcsize on python3.9 (regression from 3.8)

### ISSUE THREAD

**Issue by mikeage** [PyCQA/pylint#4297]
### Steps to reproduce

Given a file:
```python
# pylint: disable=missing-module-docstring,missing-function-docstring
import struct


def read(fmt):
    return struct.calcsize(fmt)
```

### Current behavior
(Tested on Mac Big Sur, using homebrew. Virtualenvs were used for python@3.9 and python@3.8. I see the same successful behavior on an Ubuntu 20.10 machine running python 3.8) 

```
(.venv-3.9)   0 mikemi@MCMIKEMI-32Y5M ~$ pylint struct_no_member.py
************* Module struct_no_member
struct_no_member.py:6:11: E1101: Module 'struct' has no 'calcsize' member (no-member)

--------------------------------------------------------------------
Your code has been rated at -6.67/10 (previous run: -6.67/10, +0.00)

(.venv-3.9)   2 mikemi@MCMIKEMI-32Y5M ~$ pylint --version
pylint 2.7.4
astroid 2.5.2
Python 3.9.2 (default, Mar 26 2021, 23:22:38)
[Clang 12.0.0 (clang-1200.0.32.29)]
```
And with the pre-release versions:
```
(.venv-3.9)  18 mikemi@MCMIKEMI-32Y5M ~$ pylint struct_no_member.py
************* Module struct_no_member
struct_no_member.py:6:11: E1101: Module 'struct' has no 'calcsize' member (no-member)

---------------------------------------------------------------------
Your code has been rated at -6.67/10 (previous run: -6.67/10, +0.00)

(.venv-3.9)   2 mikemi@MCMIKEMI-32Y5M ~$ pylint --version
pylint 3.0.0a1
astroid 2.5.2
Python 3.9.2 (default, Mar 26 2021, 23:22:38)
[Clang 12.0.0 (clang-1200.0.32.29)]
```

### Expected behavior
```
(.venv-3.8)   0 mikemi@MCMIKEMI-32Y5M ~$ pylint struct_no_member.py

------------------------------------
Your code has been rated at 10.00/10

(.venv-3.8)   0 mikemi@MCMIKEMI-32Y5M ~$ pylint --version
pylint 2.7.4
astroid 2.5.2
Python 3.8.9 (default, Apr  3 2021, 01:51:00)
[Clang 12.0.0 (clang-1200.0.32.29)]
```

### pylint --version output

See above

**Comment by clavedeluna** [comment]
I'm not able to reproduce this issue, which is one of those that I think when is present is very easy to reproduce, so closing. If anyone is able to repro, please re-open.

**Comment by mikeage** [comment]
As the original submitter, I can confirm that it's fixed. It was broken as late as pylint 2.11.1, and fixed in 2.12.0. (just checked)
